framework/src/test/java/org/tron/core/services/interfaceOnPBFT/http/GetEnergyPricesOnPBFTServletTest.java

```diff
@@ -0,0 +1,62 @@
+package org.tron.core.services.interfaceOnPBFT.http;
+
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+import static org.tron.common.utils.client.utils.HttpMethed.createRequest;
+
+import com.alibaba.fastjson.JSONObject;
+import java.io.IOException;
+import java.io.UnsupportedEncodingException;
+import javax.annotation.Resource;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.client.methods.HttpPost;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+import org.springframework.mock.web.MockHttpServletRequest;
+import org.springframework.mock.web.MockHttpServletResponse;
+import org.tron.common.BaseTest;
+import org.tron.core.Constant;
+import org.tron.core.config.args.Args;
+
+public class GetEnergyPricesOnPBFTServletTest extends BaseTest {
```

Comment:
Meaningless messages will be output.
<img width="658" alt="image" src="https://github.com/tronprotocol/java-tron/assets/82020050/1b0227ce-f5af-4eba-b7ce-ff2d0b588470">

Reply:
OK，I'll remove it.